tests/unittests/test_builtin_handlers.py

```diff
@@ -473,4 +474,28 @@ def test_render_jinja_payload_replaces_missing_variables_and_warns(self):
         self.assertIn(expected_log, self.logs.getvalue())
 
 
+class TestShellScriptByFrequencyHandlers(CiTestCase):
+    with_logs = True
+
+    def do_test_frequency(self, frequency):
+        from cloudinit.cmd.devel import read_cfg_paths
```

Comment:
Is there any reason these imports shouldn't be at the top of the file?

Reply:
Done (I may have been following some existing code at the time; not sure)